src/Services/SwaggerService.php

```diff
@@ -656,9 +656,52 @@ public function getDocFileContent()
             }
         }
 
+        $this->limitResponseData();
+
+        $this->cutExceptions();
```

Comment:
I believe it would be better to cut this data on the collecting documentation stage, not on presentation. Please also cover this logic by tests